app/models/grda_warehouse/project_group.rb

```diff
@@ -38,6 +38,12 @@ class ProjectGroup < GrdaWarehouseBase
       viewable_by(user)
     end
 
+    scope :text_search, -> (text) do
+      return none unless text.present?
```

Comment:
This should do some SQL sanitizing of text, not exactly sure what, but maybe just allow alpha-numeric and space - _.